flycheck.el

```diff
@@ -7051,6 +7052,17 @@ See URL `http://sass-lang.com'."
           line-end))
   :modes scss-mode)
 
+(flycheck-define-checker scss-lint
+  "A SCSS syntax checker using SCSS-Lint.
+
+See URL `https://github.com/causes/scss-lint'."
+  :command ("scss-lint" "--formatter default" source)
+  :error-patterns
+  ((error line-start (file-name) ":" line " [E] " (message) line-end)
+   (warning line-start (file-name) ":" line " [W] " (message) line-end))
+   :modes scss-mode
+   :next-checkers ((warning . scss)))
```

Comment:
Yes, that's true - we probably don't need both. I haven't used scss-lint much but it does seem far, far more detailed than the original sass/scss linter.